api_testsuite.go

```diff
@@ -227,7 +241,28 @@ func (ats *Suite) parseAndRunTest(v any, manifestDir, testFilePath string, r *re
 	loader.ServerURL = serverURL
 	loader.OAuthClient = ats.Config.OAuthClient
 
-	//Get the Manifest with @ logic
+	// Determine number of parallel repetitions, if any
+	repetitions := 1
```

Comment:
der name "repetitions" ist ungünstig. das suggeriert dass es ein "for loop" gibt der die tests einfach wiederholt. was wir hier haben ist "parallelRuns" oder sowas.